keyboards/dz60/dz60.h

```diff
@@ -384,6 +384,33 @@
     { k20,   KC_NO, k22,   k23,  k24,   k25,   k26,  k27,   k28,   k29,   k2a,   k2b,  k2c,   k2d,  KC_NO }, \
     { k30,   KC_NO, k32,   k33,  k34,   k35,   k36,  k37,   k38,   k39,   k3a,   k3b,  KC_NO, k3d,  k3e   }, \
     { k40,   k41,   KC_NO, k43,  KC_NO, KC_NO, k46,  KC_NO, KC_NO, KC_NO, KC_NO, k4b,  KC_NO, k4d,  k4e   }  \
+}^
+
+/* LAYOUT_60_olligranlund_ISO
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┐
+ * │00 │01 │02 │03 │04 │05 │06 │07 │08 │09 │0a │0b │0c │0d │0e │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┤
+ * │10   │12 │13 │14 │15 │16 │17 │18 │19 │1a │1b │1c │1d │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐2d  │
+ * │20    │22 │23 │24 │25 │26 │27 │28 │29 │2a │2b │2c │1e │    │
+ * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴────┬───┤
+ * │30  |31 │32 │33 │34 │35 │36 │37 │38 │39 │3a │3b │3d    │3e │
+ * ├────┼───┴┬──┴─┬─┴───┴───┴───┴───┴───┴──┬┴───┼───┴┬────┬────┤
+ * │40  │41  │43  │46                      │4a  │4b  │4d  │4e  │
+ * └────┴────┴────┴────────────────────────┴────┴────┴────┴────┘
+*/
+#define LAYOUT_60_olligranlund_ISO( \
```

Comment:
Is there a specific reason to add this over using the "all" layout? 

Also, we use snake case for the layout macros:
```suggestion
#define LAYOUT_60_olligranlund_iso( \
```

Reply:
Not sure how the ISO-enter is wired in the PCB, so it wasn't clear how you could achieve ISO + split backspace + split right shift. The layout I created solves this issue.